src/Game.cpp

```diff
@@ -250,13 +251,20 @@ bool Game::ProcessEvents(const SDL_Event& event)
 		case SDLK_8:
 			if ((event.key.keysym.mod & KMOD_CTRL) != 0)
 			{
-				auto handPosition = _handPose * glm::vec4(0.0f, 0.0f, 0.0f, 1.0f);
+				const auto handPosition = _handPose * glm::vec4(0.0f, 0.0f, 0.0f, 1.0f);
 				const auto index = static_cast<uint8_t>(event.key.keysym.sym - SDLK_1);
-				Locator::cameraBookmarkSystem::value().SetBookmark(index, handPosition);
+				Locator::cameraBookmarkSystem::value().SetBookmark(index, handPosition, _camera->GetOrigin());
 			}
 			else if (event.key.keysym.mod == 0)
 			{
-				// TODO(#348): Fly-to the bookmark
+				const auto& entitiesRegistry = Locator::entitiesRegistry::value();
+				const size_t index = event.key.keysym.sym - SDLK_1;
+				const auto& bookmarkEntities = Locator::cameraBookmarkSystem::value().GetBookmarks();
+				[[assume(index < bookmarkEntities.size())]];
+				const auto& entity = bookmarkEntities.at(index);
+				const auto& [transform, bookmark] =
+				    entitiesRegistry.Get<ecs::components::Transform, ecs::components::CameraBookmark>(entity);
+				_camera->GetModel().SetFlight(bookmark.savedOrigin, transform.position);
```

Comment:
Here there need to be something to make sure we don't make access on unset bookmarks (either index check use an optionnal on the saved Origin)